README.development

```diff
@@ -31,6 +31,10 @@ $ . ~/pyenv/bin/activate
 
 If the distro you are using has PyQt5 installed, make sure you have the PyQt5
 WebEngine module and development tools (eg pyqt5-dev-tools) installed as well.
+When you use your distro's PyQt5, you have to use the --system-site-packages
```

Comment:
nit: wrap --system-site-packages in backticks (\`) to get it formatted like `--system-site-packages`